lib/mm_network.pm

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/perl -w
+
+package mm_network;
+
+use base Exporter;
+use Exporter;
+
+use strict;
+
+use testapi;
+
+our @EXPORT = qw/get_host_resolv_conf configure_static_ip configure_default_gateway configure_static_dns/;
+
+sub get_host_resolv_conf {
+    my %conf;
+    open(FH, "/etc/resolv.conf");
```

Comment:
This will hit another warning from perl-critic by not using 3-argument variant of open (`open(FH, '<', '/etc/resolv.conf')`);